condor_job_metrics.py

```diff
@@ -0,0 +1,35 @@
+from prometheus_client import Counter, Histogram
+
+class JobMetrics():
+    '''
+     Wrapper class for holding prometheus job metrics
+    '''
+ 
+    def __init__(self):
+  
+        labels = ['owner','site','schedd','GPUDeviceName','usage','kind']
+        
+        memory_buckets = (1, 2, 3, 4, 6, 8, 12, 20, 40,float('inf'))
+        resource_buckets = (1, 2, 3, 4, 8, 16, float('inf'))
```

Comment:
Is it better to use `math.inf`?  That was introduced in Python 3.5.

Reply:
Supposedly  the documentation says `float('inf')` and `math.inf` are equivalent. I used `float(inf)` because what the default bucket in `client_prometheus` uses https://github.com/prometheus/client_python/blob/2ff532839fd54d7c686837e5955ea0ff9abfab76/prometheus_client/utils.py#L3

Reply:
I'm guessing they wrote that for python2 compatibility.  Either or is fine, just wanted to point it out.